### ISSUE TITLE
[BITV]: Focus won't return to the details panel after sharing details were closed 

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42958]
1. Open file details
2. Open sharing tab
3. Open sharing details "Open Sharing Details"
4. Navigate through the items, click to "update share" or "cancel"
5. Next focus stop will start on the skip links (start of the page), but should be a next interactive element in a sharing sidebar

![Peek 2024-01-19 11-28](https://github.com/nextcloud/server/assets/6078378/e330b6b6-99a2-4728-8951-bb5effd26dd0)
